### ISSUE TITLE
studio(a11y): main menu not accessible with keyboard navigation only

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#605]
### Affected part of DeckDeckGo
- [x] Studio
- [ ] Remote control
- [ ] Documentation
- [ ] Demo
- [ ] Starter kit
- [ ] Poll
- [ ] Web Components
- [ ] Templates

### Reproduction

Steps to reproduce:
1. Go to https://deckdeckgo.com/
2. Navigate with tab once (main menu is focused)
3. Press Enter
4. Tab once again
 
### Expected Behavior

Menu items like _Home_ should be focusable.

### Actual Behavior

The overlay for Main menu is not focusable. Tabbing through the content continues behind overlay.

![Tabbing through DeckDeckGo content](https://user-images.githubusercontent.com/1763537/73140484-b891ba00-4079-11ea-8acb-64c7b3aa2485.gif)
_The current behaviour in action. (I needed to click once to get focus inside the browser, the rest is done with tab key only)_

### Environment

- Browser(s): Chrome
- Operating System (e.g. Windows, macOS, Ubuntu): macOS

**Comment by peterpeterparker** [comment]
It looks like I don't use that much often "tab" ;)

Thx for the issue @donroyco 👍 

I link this one with https://github.com/deckgo/deckdeckgo/issues/534, kind of the same idea

**Comment by peterpeterparker** [comment]
@donroyco just had a quick test. actually, at least in the next version, the items in the menu are accessible through tabs (see gif) but the things is that the menu/popover receive the focus/tabs after the homepage has been tabbed.

any idea how could the focus/tabs could be shifted to give the priority to the opened menu?

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/16886711/75046746-ef66be80-54c5-11ea-9aa4-5016c0ec2f70.gif)

**Comment by donroyco** [comment]
Not sure how this would work in Ionic, I don't see some a11y documentation on [ionic-menu ](https://ionicframework.com/docs/api/menu). Is there a way to manually focus the sidebar when you know it has been triggered to open?

**Comment by peterpeterparker** [comment]
@donroyco thx for the feedback! I honestly don't know neither and didn't found anything about it.

Therefore I opened a feature request in the Ionic Core https://github.com/ionic-team/ionic/issues/20590

**Comment by donroyco** [comment]
@peterpeterparker Cool. I subscribed to the issue. Strange this is not available already in Ionic.

In Angular Material you [can check it pretty easily](https://stackblitz.com/edit/angular-xqu6vg?file=src/app/sidenav-open-close-example.html).
